R/S3.R

```diff
@@ -2314,3 +2314,34 @@ pander.summary.rms <- function (x, ...) {
     cat('\n')
     invisible()
 }
+
+#' Prints an ets object from forecast package in Pandoc's markdown.
+#' @param x an ets object
+#' @param digits number of digits of precision
+#' @param ... optional parameters passed to raw \code{pandoc.table} function
+#' @export
+pander.ets <- function(x, digits = panderOptions('digits'),...) {
+  cat('\nCall:', pandoc.formula.return(x$call), '', sep = '\n')
+  cat('Type of ets: ', x$method, '\n', sep = '')
+  lambda<-x$lambda
+  isn <- names(x$initstate)
+  initstate <- matrix(x$initstate, nrow = 1)
+  colnames(initstate) <- isn
+  sp<-x$par["alpha"]
+  if (x$components[2] != "N")
```

Comment:
please put `if` body in `{`, since `covr` can't handle `if` statements without braces

if (x$components[3] != "N") {
    sp<-c(sp, x$par["gamma"])
}